webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js

```diff
@@ -141,7 +141,7 @@ describe('Test utils', () => {
     });
   });
 
-  it('Check that all test specs belong to a test suites', async () => {
+  it.skip('Check that all test specs belong to a test suites', async () => {
```

Comment:
@kapelo  Please add a note in one of the tickets tracking e2e work to re-enable this test.

Reply:
Ok @Benmuiruri